cmd/cli/plugin/secret/main.go

```diff
@@ -16,14 +16,13 @@ import (
 )
 
 var descriptor = cliv1alpha1.PluginDescriptor{
-	Name:        "imagepullsecret",
-	Description: "Manage image pull secret operations. Image pull secrets enable the package and package repository consumers to authenticate to private registries.",
+	Name:        "secret",
+	Description: "Tanzu secret management",
```

Comment:
if it should hint about the rename in the description and any instructions for user to clean up the old plugin ?(same for README)